aix.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+const getSqlStatement = family => {
+  let sql =
+    "select NEXT_HOP, LOCAL_BINDING_INTERFACE from QSYS2.NETSTAT_ROUTE_INFO where ROUTE_TYPE='DFTROUTE' and NEXT_HOP!='*DIRECT'";
+
+  if (family === "v4") {
+    sql += " and CONNECTION_TYPE='IPV4'";
+  } else {
+    sql += " and CONNECTION_TYPE='IPV6'";
+  }
+
+  return sql;
+};
+
+const getGatewayInformationAsync = async family => {
+  return new Promise((resolve, reject) => {
+    try {
+      const idbConnector = require("idb-connector");
+
+      const dbconn = new idbConnector.dbconn();
+      dbconn.conn("*LOCAL");
+
+      const sql = getSqlStatement(family);
+      const stmt = new idbConnector.dbstmt(dbconn);
+
+      stmt.exec(sql, async results => {
+        try {
+          stmt.close();
+          dbconn.disconn();
+          dbconn.close();
+        } catch (err) {
+          reject(new Error("Unable to determine default gateway"));
+          return;
+        }
+
+        if (results && results[0] && results[0].NEXT_HOP) {
+          resolve({
+            gateway: results[0].NEXT_HOP,
+            interface: results[0].LOCAL_BINDING_INTERFACE || ""
```

Comment:
Does the query return a actual `null` for `LOCAL_BINDING_INTERFACE` in the IPv6 case?

Reply:
Since it‘s a system table, it might be null. That‘s why I put it like this, just in case.

Reply:
Yes, but I want to know if the `null` in the docs transform to a actual JavaScript `null` value or maybe `'null'` or other weird things.

Reply:
Oh, this is what you mean.
According to the source of idb-connector dbstmt.cc ist translates `SQL_NULL_DATA` to an actual `null` value
See [dbstmt.cc](https://bitbucket.org/litmis/nodejs-idb-connector/src/318f12eecda1120839054d76b805a927cd172c20/src/db2ia/dbstmt.cc?fileviewer=file-view-default#lines-344) lines 344-345

Reply:
Thanks for digging that out!